src/worker/abstract-worker.ts

```diff
@@ -2,22 +2,42 @@ import { AsyncResource } from 'async_hooks'
 import type { MessageValue } from '../utility-types'
 import type { WorkerOptions } from './worker-options'
 
+/**
+ * Basic worker abstraction used for common logic between poolifier workers.
+ *
+ * @template MainWorker Type of main worker.
+ * @template Data Type of data this worker receives from pool's execution.
+ * @template Response Type of response the worker sends back to the main worker.
+ */
 export abstract class AbstractWorker<
   MainWorker,
   Data = unknown,
   Response = unknown
 > extends AsyncResource {
+  /**
+   * Maximum time this worker can be alive. The pool will check and terminate this worker when the time expires.
+   */
   protected readonly maxInactiveTime: number
+  /**
+   * Whether the worker is working asynchronously or not.
+   */
   protected readonly async: boolean
+  /**
+   * Timestamp of the last task processed by this worker.
```

Comment:
Last task queued or last time finished processing a task?

Reply:
The code says: Last time a message was send to the main worker :thinking:  
@pioardi Could you help here? Maybe you know your project better :slightly_smiling_face:

Reply:
This property indicate when the last task was processed by this worker.
If you submit a task at 11:00:02 and the tasks finish at 11:00:04, this variable value will be 11:00:04 for the worker that worked on the task